src/wp-admin/js/plugin-install.js

```diff
@@ -226,4 +226,12 @@ jQuery( function( $ ) {
 				$uploadViewToggle.attr( 'aria-expanded', $body.hasClass( 'show-upload-view' ) );
 			});
 	}
+
+	/* Plugin install Category filter JS */
+	$( '.plugin-categories-filter a' ).click( function( event ) {
```

Comment:
`jQuery.fn.click() event shorthand is deprecated`
Should we fix this as we are changing the line?

Reply:
Agreed `.click(` updated to `.on( 'click',`